README.md

```diff
@@ -38,8 +38,8 @@ a Kubernetes cluster.
 
 ### Kubernetes 1.29.0 or Later
 
-Starting with Operator v7.1.0, MinIO requires Kubernetes version 1.29.0 or later. You must upgrade your Kubernetes
-cluster to 1.29.0 or later to use Operator v7.1.0+.
+Starting with Operator v7.1.1, MinIO requires Kubernetes version 1.29.0 or later. You must upgrade your Kubernetes
```

Comment:
Is it like we need k8s 1.29.0 starting v7.1.1 of operator and older ones can work with older versions of k8s?

Reply:
I think this is a replace-all issue.

Reply:
The requirement has been allways to upgrade kubernetes to a maintained version @shtripat, on this release I updated the version on which the tests are running to k8s 1.30.0 and above.